security/libp11/Portfile

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
+
+name                    libp11
+github.setup            OpenSC libp11 0.4.10 libp11-
+github.tarball_from     releases
+revision                1
+categories              security
+platforms               darwin
+license                 LGPL-2.1
+maintainers             {mail.ru:nano103 @nano103} openmaintainer
+
+description             Interface to access PKCS#11 objects.
+long_description        libp11 is a library implementing a thin layer on top of PKCS#11 API \
+                        to make using PKCS#11 implementations easier.
+
+checksums               rmd160  3a1368ff10f8481ee6a49812ceabd9877a171436 \
+                        sha256  639ea43c3341e267214b712e1e5e12397fd2d350899e673dd1220f3c6b8e3db4 \
+                        size    496891
+
+distname                ${github.project}-${github.version}
```

Comment:
The `distname` should be correctly set by the GitHub portgroup. So this line can be removed.